packages/gatsby/src/utils/api-runner-error-parser.ts

```diff
@@ -1,21 +1,23 @@
-const errorParser = ({ err }) => {
+import { IMatch } from "../types"
+
+const errorParser = ({ err }): IMatch => {
   const handlers = [
     {
       regex: /(.+) is not defined/m,
-      cb: match => {
+      cb: (match): IMatch => {
         return {
           id: `11330`,
-          context: { message: match[0], arg: match[1] },
+          context: { sourceMessage: match[0], arg: match[1] },
```

Comment:
`message` was inconsistent here (The IMatch interface unconvered that) so it was changed to `sourceMessage` - also in error-map